llvm/test/CodeGen/X86/fake-use-remove-loads.mir

```diff
@@ -1,7 +1,8 @@
 # NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
 # Ensure that loads into FAKE_USEs are correctly removed by the
 # remove-loads-into-fake-uses pass.
```

Comment:
"... and that if we do not use instruction referencing, no modifications are made"